TriblerGUI/widgets/trustpage/js/radial_view/radial_inspector.js

```diff
@@ -38,15 +38,19 @@ function RadialInspector(d3element, options) {
             bindings = {
                 focusNodeName: self._getNodeIdentifier(data.focus_node),
                 nodeCount: data.nodes.length,
-                linkCount: data.links.length
+                linkCount: data.links.length,
+                color: self.config.node.color.range[0]
             };
 
         self._setContents(
             "Tribler Trust Network",
             "Partial view around {focusNodeName}",
             [
-                "Showing <strong>{nodeCount} users</strong>.",
-                "Click on users to explore the rest of the network."
+                "Showing <strong>{nodeCount} users</strong>",
+                "Click on users to explore the rest of the network.",
```

Comment:
Please take singular/plural noun (user/users) into account. Moreover, when `{nodeCount}` equals 1, it is not possible to click on users to explore the rest of the network, and should therefore not be shown